pom.xml

```diff
@@ -160,11 +160,6 @@
                         <artifactId>surefire-junit-platform</artifactId>
                         <version>${dep.plugin.surefire.version}</version>
                     </dependency>
-                    <dependency>
-                        <groupId>org.apache.maven.surefire</groupId>
-                        <artifactId>surefire-testng</artifactId>
-                        <version>${dep.plugin.surefire.version}</version>
-                    </dependency>
```

Comment:
have to get rid of testng from parent pom otherwise gateway-ha will inherit and throw exception because we removed all testng related deps from gateway-ha